secretstores/local/keyring/keyring.go

```diff
@@ -0,0 +1,192 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package keyring
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/99designs/keyring"
```

Comment:
It looks like this depends on `github.com/99designs/go-keychain` which requires CGO.

Can you confirm that this PR works **without CGO** (ie. when building and compiling with `CGO_ENABLED=0`) on all 3 platforms? Sadly, using CGO is not possible with Dapr.

Reply:
My fault, I didn't notice that Dapr doesn't support CGO. It seems that except for backends of file and pass,  all require CGO. So this feature, local keyring secret store, should not be needed?